arex-storage-web-api/src/main/java/com/arextest/storage/web/controller/AgentRecordingController.java

```diff
@@ -96,6 +104,38 @@ public byte[] query(@RequestBody AREXMocker requestType,
     return ZstdJacksonSerializer.EMPTY_INSTANCE;
   }
 
+  @PostMapping(value = "/queryAllMocker")
+  @ResponseBody
+  public byte[] queryAllMocker(@RequestBody QueryMockRequestType requestType) {
+    try {
+      if (StringUtils.isEmpty(requestType.getRecordId())) {
+        LOGGER.warn("agent query recordId empty");
+        return ZstdJacksonSerializer.EMPTY_INSTANCE;
+      }
+
+      MDCTracer.addRecordId(requestType.getRecordId());
+      String[] categoryTypes = requestType.getCategoryTypes();
+      if (ArrayUtils.isEmpty(categoryTypes)) {
+        LOGGER.warn("agent query categoryTypes empty, recordId: {}", requestType.getRecordId());
+        return ZstdJacksonSerializer.EMPTY_INSTANCE;
+      }
+
+      return agentWorkingService.queryAllMocker(requestType.getRecordId(), requestType.getFieldNames(), requestType.getCategoryTypes());
```

Comment:
suggest add queryAllMocker cost

Reply:
arex-common sdk has unified statistics of time-consuming operations